src/index.ts

```diff
@@ -1,26 +1,144 @@
+import moment from 'moment';
+import { Request, Response } from 'express';
 import fs from "fs";
 import { cwd } from "process";
 import { Environments } from "./enums/environments";
+import { get, postWithConfig } from './utils/request';
 
 const DEFAULT_CONFIG_FILE_LOCATION = `${cwd()}/.bluebutton-config.json`;
 const SANDBOX_BASE_URL = "https://sandbox.bluebutton.cms.gov";
 const PRODUCTION_BASE_URL = "https://api.bluebutton.cms.gov";
+const GENERAL_DATA_ERR = '{"message": "Unable to load data - query FHIR resource error."}';
 
+type CodeChallenge = {
```

Comment:
Your PR shouldn't have any need of codeChallenge/verifier. That work is left up to @dtisza1 's PR.

Reply:
Good catch, some leftover code, will remove.